server.js

```diff
@@ -42,10 +45,12 @@ function startGame() {
     if (users.count() >= MIN_PLAYER_COUNT) { // if nobody has left lobby
       io.to(currentRoom).emit('stateChange', 'game')
       console.log(`${currentRoom} started a game!`)
+
       assignRoles(currentRoom)
+
+      roundManager.start(io, currentRoom, users)
       currentRoom = generateRoomName()
-      users.clear()
-      io.emit('lobbyUpdate', users.list())
+      io.emit('lobbyUpdate', users.names())
```

Comment:
I'm not sure what it's doing... I don't think it ever had to be here, even when the `clear()` was still present